lib/hoardable/engine.rb

```diff
@@ -81,6 +81,16 @@ def at(datetime)
       @at = nil
     end
 
+    # Allows calling code to set the upper bound for the temporal range for recorded audits.
+    #
+    # @param datetime [DateTime] the datetime to temporally record audits at
```

Comment:
```suggestion
    # @param datetime [DateTime] the datetime to temporally record versions at
```

just a wording suggestion for consistency